fast64_internal/f3d/f3d_material.py

```diff
@@ -1428,6 +1455,12 @@ def update_all_node_values(material, context):
     update_rendermode_preset(material, context)
 
 
+def update_all_material_nodes(self, context):
+    for material in bpy.data.materials:
+        with context.temp_override(material=material):
+            update_all_node_values(material, context)
```

Comment:
this is a bit too trigger-happy, maybe at least check the material is a f3d material before putting it through this